Dockerfile

```diff
@@ -18,22 +18,20 @@ RUN \
     /usr/share/selkies/www/icon.png \
     https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/remmina-logo.png && \
   echo "**** install remmina ****" && \
-  if [ -z "${REMMINA_RELEASE}" ]; then \
-    REMMINA_RELEASE=$(curl -sX GET http://archive.ubuntu.com/ubuntu/dists/noble-updates/main/binary-amd64/Packages.gz | gunzip |grep -A 7 -m 1 'Package: remmina' | awk -F ': ' '/Version/{print $2;exit}'); \
-  fi && \
+  apt-add-repository ppa:remmina-ppa-team/remmina-next && \
   apt-get update && \
   apt-get install -y \
     librsvg2-common \
-    remmina="${REMMINA_RELEASE}" \
-    remmina-plugin-exec="${REMMINA_RELEASE}" \
-    remmina-plugin-kiosk="${REMMINA_RELEASE}" \
-    remmina-plugin-kwallet="${REMMINA_RELEASE}" \
-    remmina-plugin-rdp="${REMMINA_RELEASE}" \
-    remmina-plugin-secret="${REMMINA_RELEASE}" \
-    remmina-plugin-spice="${REMMINA_RELEASE}" \
-    remmina-plugin-vnc="${REMMINA_RELEASE}" \
-    remmina-plugin-www="${REMMINA_RELEASE}" \
-    remmina-plugin-x2go="${REMMINA_RELEASE}" && \
+    remmina \
+    remmina-plugin-exec \
+    remmina-plugin-kiosk \
+    remmina-plugin-kwallet \
+    remmina-plugin-rdp \
+    remmina-plugin-secret \
+    remmina-plugin-spice \
+    remmina-plugin-vnc \
+    remmina-plugin-www \
+    remmina-plugin-x2go && \
```

Comment:
why do we not want to assert the version?

Reply:
More here https://github.com/linuxserver/docker-mediaelch/pull/10, my logic is this assures very little and adds more maintenance overhead. I am not going to die on this hill just trying to simplify things. If it is a must I will revert. Specifcally for the plugins with the ppa you can't pin to the ppa version so it would just be remmina in this case.

Reply:
Asserting it makes sense for when other arches takes time to build. We advertise a app version, if we end up pushing the wrong version, that doesn't look good. 
Provided that our package check only runs on amd64, this is not something that would be caught by that check, and therefore rely on only other os package versions to change in order to rectify the discrepancy. 

Asserting only the main app makes sense.